code/stable/projectile/projectile_u_p_l_b_b_c_d/src/cpp/Projectile.hpp

```diff
@@ -48,32 +48,6 @@ class Constants {
         
 };
 
-/** \brief Calculates flight duration: the time when the projectile lands (s)
```

Comment:
When comments are turned on, I think a version of these comments should go above the inlined code.